include/mega/megaclient.h

```diff
@@ -224,6 +224,9 @@ class MEGA_API MegaClient
     // queue a user attribute retrieval (for non-contacts)
     void getua(const char* email_handle, const attr_t at = ATTR_UNKNOWN, int ctag = -1);
 
+    // retrieve the email address of a user
+    void getUserEmail(const char *uid = NULL);
```

Comment:
I think that the default parameter should be removed. Since it's not checked if the value is NULL, the SDK will probably crash. Also, having it doesn't add any advantage because the function can't succeed without it. It would be different if it could provide the own email when NULL is passed, for example.